.cirrus.yml

```diff
@@ -126,10 +126,11 @@ cross_build_task:
 
     osx_instance:
         image: 'big-sur-base'
-
+    env:
+        PATH: "/usr/local/opt/go@1.18/bin:$PATH"
     script:
         - brew update
-        - brew install go
+        - brew install go@1.18
```

Comment:
I'm always hesitant to tag a particular version in a file like this.....  Can we just stick with latest?  I'm just concerned later down the line when we're looking at go@1.25 or what have you.

Reply:
The problem with latest is that certain tools we rely on like lint, are not updated to use it and cause unfixable failures in CI/CD systems.

Reply:
Since this is a branch, I see no reason not to freeze it at the golang that was used originally.

Reply:
Yep, @TomSweeneyRedHat I would agree with you completely were this the 'main' branch.  For releases, we really want to lock them down to support future backports.  Otherwise every release-branch is a moving-target :scream: